sourcegraph/about
# Update CodeInsightSearchExample component styles

Closes #5137 by setting `height` on the `CodeInsightSearchExample` component.

## To Test
- Go to the `code-insights` demo page.
- Check that both charts look okay on Safari and other browsers, and on mobile
